docs/getting-started.rst

```diff
@@ -26,16 +25,29 @@ First you need to get web3.js into your project. This can be done using the foll
 
 - npm: ``npm install web3``
 - meteor: ``meteor add ethereum:web3``
-- pure js: link the ``dist/web3.min.js``
 
 After that you need to create a web3 instance and set a provider.
-Ethereum supported Browsers like Mist or MetaMask will have a ``ethereumProvider`` or ``web3.currentProvider`` available. For  web3.js, check ``Web3.givenProvider``.
-If this property is ``null`` you should connect to a remote/local node.
+Ethereum supported Browsers like Mist or MetaMask will have a ``ethereum`` or ``web3.currentProvider`` available.
+For  web3.js, check ``Web3.givenProvider``. If this property is ``null`` you should connect to your own local or remote node.
```

Comment:
For L30-L31, maybe make explicit MetaMask isn't a browser (yet), and that ethereum and currentprovider are window properties.

Browsers with Ethereum support, such as Mist and FireFox with Metamask, have the a window property called either ``ethereum`` or ``web3.currentProvider``.

With the web3.js library you can retrieve the browser's provider with ``Web3.givenProvider``. In case this property is ``null``, you need to connect to a node, local or remote.